src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -161,47 +132,11 @@ const personalDetailsSelector = (personalDetails) =>
  * use it to prevent re-renders from parent re-renders.
  */
 export default React.memo(
-    compose(
-        withOnyx({
-            comment: {
-                key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${reportID}`,
-            },
-            fullReport: {
-                key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
-            },
-            reportActions: {
-                key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`,
-                canEvict: false,
-            },
-            personalDetails: {
-                key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-                selector: personalDetailsSelector,
-            },
-            preferredLocale: {
-                key: ONYXKEYS.NVP_PREFERRED_LOCALE,
-            },
-        }),
-        // eslint-disable-next-line rulesdir/no-multiple-onyx-in-file
-        withOnyx({
-            parentReportActions: {
-                key: ({fullReport}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${fullReport.parentReportID}`,
-                canEvict: false,
-            },
-            policy: {
-                key: ({fullReport}) => `${ONYXKEYS.COLLECTION.POLICY}${fullReport.policyID}`,
-            },
-            // Ideally, we aim to access only the last transaction for the current report by listening to changes in reportActions.
-            // In some scenarios, a transaction might be created after reportActions have been modified.
-            // This can lead to situations where `lastTransaction` doesn't update and retains the previous value.
-            // However, performance overhead of this is minimized by using memos inside the component.
-            receiptTransactions: {key: ONYXKEYS.COLLECTION.TRANSACTION},
-        }),
-        // eslint-disable-next-line rulesdir/no-multiple-onyx-in-file
-        withOnyx({
-            transaction: {
-                key: ({fullReport, parentReportActions}) =>
-                    `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(parentReportActions, [fullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'], '')}`,
-            },
-        }),
-    )(OptionRowLHNData),
+    withReportCommentDrafts({
```

Comment:
*question:* If there's a reason to use it at the row level, can we use a selector so that the row will only re-render if a draft comment related to the report for that row changes?

Reply:
replacement of `withReportCommentDrafts` is already merged to main - https://github.com/Expensify/App/pull/30031/files#r1377405180

Reply:
but as Rory suggested, that also should be moved to `LHNOptionsList` if possible

Reply:
I removed `withReportCommentDrafts` from row-level and passed it from `LHNOptionList` as suggested 👍